File: problemas/iniciante/1144/1144.lua
n = tonumber(io.read(), 10)
for i = 1, n do
  print(string.format("%d %d %d", i, i * i, i * i * i))
  print(string.format("%d %d %d", i, i * i + 1, i * i * i + 1))
end

File: problemas/iniciante/1144/1144.clj
(defn leitura [i n]
  (if (<= i n) (do (printf "%d %d %d\n" i (* i i) (* i i i)) (printf "%d %d %d\n" i (+ (* i i) 1) (+ (* i i i) 1)) (leitura (inc i) n)) ()))

(def n (read-string (read-line)))
(leitura 1 n)